batch/src/main/java/com/ImSnacks/NyeoreumnagiBatch/BatchConfiguration.java

```diff
@@ -0,0 +1,59 @@
+package com.ImSnacks.NyeoreumnagiBatch;
+
+import org.springframework.batch.core.configuration.annotation.EnableBatchProcessing;
+import org.springframework.batch.core.repository.JobRepository;
+import org.springframework.batch.core.repository.support.JobRepositoryFactoryBean;
+import org.springframework.batch.item.database.support.DataFieldMaxValueIncrementerFactory;
+import org.springframework.batch.item.database.support.DefaultDataFieldMaxValueIncrementerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.support.incrementer.DataFieldMaxValueIncrementer;
+import org.springframework.jdbc.support.incrementer.SqlServerMaxValueIncrementer;
+import org.springframework.transaction.PlatformTransactionManager;
+
+import javax.sql.DataSource;
+
+@Configuration
+@EnableBatchProcessing
+public class BatchConfiguration {
+
+  private static final String ISOLATION_REPEATABLE_READ = "ISOLATION_READ_COMMITTED";
```

Comment:
P1: 변수명 이렇게 고쳐야할 것 같아요~~

```suggestion
  private static final String ISOLATION_READ_COMMITTED = "ISOLATION_READ_COMMITTED";
```